py_termianl_graph/py_terminal_graph.py

```diff
@@ -0,0 +1,131 @@
+		self.width, self.height = get_terminal_size()
+		self.width -= 1
+
+		# Creating matrix for screen
+		self.screen = [list(" " * self.width) for _ in range(self.height)]
+
+
+	def draw(self, color : str = ""):
+		"""
+			Process and draw an image
+		"""
+		self.get_image()
+		self.render()
+
+	def get_image(self):
+		"""
+			Draws all art to screen matrix
+		"""
+
+		from textwrap import TextWrapper  # Necessary for wrapping lines
+
+		left_margin = len(str(max([round(a, 5) for a, _ in self.data]))) + 1 # Distance between left edge and vertical line
+		bottom_margin = 5 # Distance between bottom corner and horizontal line (from bottom to top)
+
+		max_x = max([x for x, y in self.data])
+		max_y = max([y for x, y in self.data])
+
+		# ==== Operating Data =====
+		prev_pos = (left_margin, self.height - bottom_margin)
+		for num, (x, y) in enumerate(self.data): # Loop through all items in data
+			x_pos = left_margin + int(x / max_x * (self.width - left_margin - 1))
+			y_pos = self.height - bottom_margin - int(y / max_y * (self.height - bottom_margin))
+			pos = (x_pos, y_pos)
+
+			self.line(prev_pos, pos, color=self.Colors.DARK_GREY)
+
+			prev_pos = pos
+
+			self.screen[pos[1]][pos[0]] = self.color + "█" # Paint point with solid
+			# Record value (Left)
+			str_value = str(round(y, 5)).rjust(left_margin, " ") # Fill spaces from left edge (margin word to the right)
+			for i in range(left_margin):
+				self.screen[pos[1]][i] = self.color + str_value[i] # Copy word char by char
+
+			# Record name (Bottom)
+			# TextWrapper needed to automatically wrap lines
+			# max lines = 4
+			# long words will be shortened
+			# if name is too long it will become "..."
+			# max width = column_width
+			for i in range(len(str(x))):
+				if str(x)[i] != " ":
+					self.screen[self.height - bottom_margin + num % bottom_margin]\
+								[x_pos - len(str(x)) // 2 + i] = str(x)[i]
+
+		# Vertical line
+		for y in range(self.height - bottom_margin):  # From top edge to bottom corner (bottom margin)
+			self.screen[y][left_margin] = self.color + "│"  # Set vertical line symbol
+		# Horizontal bar
+		for x in range(left_margin + 1, self.width):  # From left corner (left margin) to right edge
+			self.screen[self.height - bottom_margin][x] = self.color + "─"  # Set horizontal line symbol
+		# Corner
+		self.screen[self.height - bottom_margin][left_margin] = self.color + "└"  # Set top-right corner symbol
+
+	def render(self):
+		"""
+			Renders image from self.screen using prints
+		"""
+
+		from os import system, name
+		system("cls" if name == "nt" else "clear")
+
+		for y in range(self.height):
+			print("".join(self.screen[y]))
+		# input("Press any key to continue... ")
+
+
+if __name__ == "__main__":
+	# Fills data with values
+
+	from math import sin
+	data = [(x, abs(sin(x / 20))) for x in range(0, 100)]
+
+	diagram_static = GraphStatic(data, color=GraphStatic.Colors.GREEN) # Creating object for graph
+	diagram_static.draw() # Draw graph
```

Comment:
PEP 8: E261 at least two spaces before inline comment.
```suggestion
	diagram_static = GraphStatic(data, color=GraphStatic.Colors.GREEN)  # Creating object for graph
	diagram_static.draw()  # Draw graph
```